CONTRIBUTION.md

```diff
@@ -0,0 +1,83 @@
+# Contribution To The Project
+
+Pull Requests and Issues have four possible Labels:
+* WIP - Work In Progress, it is not ready for review/merge
+* Functional - Proposed change changes the test behaviour
+* Metadata - Only Metadata is about to change
+* Scope - Big scope, a commit changes multiple test scenarios
+
+Rules:
+* A commit introduces one logical unit and it should not alter multiple testcases
+* Do not merge your own Pull Request
+
+
+Workflow:
+* Create a fork
+* Clone a fork
+* Add remote to your fork, call it upstream
```

Comment:
Perhaps be more clear about upstream, something like this _Add this repository as a remote of your fork, call it upstream, e.g:_

Reply:
I used your sentence.